src/commands/cmd_server.cc

```diff
@@ -815,6 +818,11 @@ class CommandHello final : public Commander {
     } else {
       output_list.push_back(redis::BulkString("standalone"));
     }
+    output_list.push_back(redis::BulkString("role"));
+    output_list.push_back(redis::BulkString(srv->IsSlave() ? "slave" : "master"));
+    // For Kvrocks, the modules is not supported.
+    output_list.push_back(redis::BulkString("modules"));
+    output_list.push_back(conn->NilArray());
```

Comment:
I don't think it should be an empty array. We DO support some redis modules.

Reply:
@PragmaTwice Yes, we did support some modules like JSON, but the mechanism is different from the Redis modules. For the Redis HELLO command, the module field needs to return the module library path and integer version which NOT exist in Kvrocks, so I kept this as an empty array.

```
   1) 1# "name" => "ReJSON"
      2# "ver" => (integer) 20606
      3# "path" => "/opt/redis-stack/lib/rejson.so"
      4# "args" => (empty array)
```

Reply:
Ahh I see.